### ISSUE TITLE
Warning: Remote connection disconnect.

### ISSUE THREAD

**Issue by kkamkou** [Letscodeit/docker-dev-image#1]
``` sh
Bringing machine 'default' up with 'virtualbox' provider...
==> default: Checking if box 'hash/universal-docker' is up to date...
==> default: Clearing any previously set forwarded ports...
==> default: Clearing any previously set network interfaces...
==> default: Preparing network interfaces based on configuration...
    default: Adapter 1: nat
==> default: Forwarding ports...
    default: 22 => 2222 (adapter 1)
==> default: Booting VM...
==> default: Waiting for machine to boot. This may take a few minutes...
    default: SSH address: 127.0.0.1:2222
    default: SSH username: vagrant
    default: SSH auth method: private key
    default: Warning: Connection timeout. Retrying...
    default: Warning: Remote connection disconnect. Retrying...
    default: Warning: Remote connection disconnect. Retrying...
    default: Warning: Remote connection disconnect. Retrying...
    default: Warning: Remote connection disconnect. Retrying...
    default: Warning: Remote connection disconnect. Retrying...
    default: Warning: Remote connection disconnect. Retrying...
    default: Warning: Remote connection disconnect. Retrying...
```

``` sh
$ vagrant ssh-config
Host default
  HostName 127.0.0.1
  User vagrant
  Port 2222
  UserKnownHostsFile /dev/null
  StrictHostKeyChecking no
  PasswordAuthentication no
  IdentityFile "C:/Users/MY_ACCOUNT/.vagrant.d/boxes/hash-VAGRANTSLASH-universal-docker/1.1/virtualbox/vagrant_private_key"
  IdentitiesOnly yes
  LogLevel FATAL
```

**Comment by kkamkou** [comment]
fixed by: disconnect network cable at vb and connect afterwards

**Comment by kkamkou** [comment]
https://github.com/mitchellh/vagrant/issues/3860
https://bugs.launchpad.net/cloud-init/+bug/1315501

**Comment by kkamkou** [comment]
At the end of `/etc/networking/interfaces` file I had `Vagrant` stuff. After removal evth. works as expected

**Comment by hschaeidt** [comment]
I had the same issue with the prebuilt image. I will check if this is due to building error.

**Comment by hschaeidt** [comment]
Fixed in box version v0.0.2
